### PR TITLE
Updates to Consumer/Producer, optional parsing of msg.value

### PR DESCRIPTION
**Pull request by amanelis**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,7 +52,7 @@ install: clean venv deps
 	. venv/bin/activate; python setup.py install
 
 lint:
-	. venv/bin/activate; pylint cybric --output-format=colorized --disable=W1234 --ignore build,dist,htmlcov,stage,tests,venv,*.egg-info
+	. venv/bin/activate; python -m flake8 --ignore=E501 cybric/
```
Comment (JohnDzialo) [inline Makefile]: what up with this?

Comment (JohnDzialo) [inline Makefile]: flake over pylint?

File: cybric/events/consumer.py [hunk 2]
```diff
@@ -12,22 +12,20 @@ def __init__(self, config, options={}):
...
-                self.config.logger.error(f'Failed to consume event: {err}')
+                self.config.logger.error(f'Failed to consume event: KafkaError[{err}]')
             else:
-                self.msg = await self.decode_message(msg_bytes=raw_msg.value)
-                self.payload = self.msg.get('payload', None)
-                self.config.logger.info(f'Consumed Msg: {self.msg}')
-                await q.put(self.msg)
+                if decode:
+                    msg = await self.decode_message(msg_bytes=msg.value)
+                    self.config.logger.info(f'Consumed msg: {self.msg}')
```
Comment (JohnDzialo) [inline cybric/events/consumer.py]: self.msg no longer exists, should be msg??

File: cybric/events/consumer.py [hunk 3]
```diff
@@ -12,22 +12,20 @@ def __init__(self, config, options={}):
...
+                msg = await consumer.getone()
             except KafkaError as err:
-                self.config.logger.error(f'Failed to consume event: {err}')
+                self.config.logger.error(f'Failed to consume event: KafkaError[{err}]')
             else:
-                self.msg = await self.decode_message(msg_bytes=raw_msg.value)
-                self.payload = self.msg.get('payload', None)
-                self.config.logger.info(f'Consumed Msg: {self.msg}')
-                await q.put(self.msg)
+                if decode:
```
Comment (JohnDzialo) [inline cybric/events/consumer.py]: I think we should always decode the msg, the bool argument should either return decoded msg with metadata not decoded msg.value which only includes the kafka message no metadata

File: cybric/messages/base.py [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
+    DRYRUN = 'dryrun'
+    JOB_ID = 'jobId'
+    ID = 'id'
+    IS_VALIDATED = 'is_validated'
+    MESSAGE = 'message'
+    PAYLOAD = 'payload'
+    POLICY = 'policy'
+    POLICY_ID = 'policyId'
+    RESPONSE = 'response'
+    TYPE = 'type'
```
Comment (JohnDzialo) [inline cybric/messages/base.py]: i almost want to just get rid of the MsgLib completely...  but it's fine for now

File: Makefile (hunk 1 above)
Comment (amanelis) [inline Makefile]: pylint 3.6 gave a bunch of errors, was able to produce better results using flake8

File: cybric/events/consumer.py (hunk 2 above)
Comment (amanelis) [inline cybric/events/consumer.py]: good catch.

**Pull request merged**